model/project.rb

```diff
@@ -5,6 +5,8 @@
 class Project < Sequel::Model
   one_to_many :access_tags
   one_to_many :access_policies
+  one_to_many :access_control_entries
+  one_to_many :subject_tags
```

Comment:
We might include the commit that adds also `object_tags`, `action_tags` to this PR too.

Reply:
That makes sense.  I updated the last commit in this PR to do that.